impl/src/main/resources/com/okta/authn/sdk/config/okta.yaml

```diff
@@ -25,3 +25,7 @@ okta:
       host:
       username:
       password:
+    retry:
+    requestTimeout: 10
```

Comment:
This point got lost in recent conversations (and the current wiki spec), but I do still think it's good to have the authN libraries timeout after 10 seconds.  FYI @nbarbettini and @rchild-okta. Unless there is disagreement, let's patch the spec to have this again.

Reply:
We're talking about defaulting to 10s right? Yep, agree.